src/main/java/nl/novi/TechItEasy/Models/Television.java

```diff
@@ -0,0 +1,202 @@
+package nl.novi.TechItEasy.Models;
+
+import jakarta.persistence.*;
+import java.util.List;
+
+@Entity
+@Table
+public class Television {
+    @Id
+    @GeneratedValue
+    private Long id;
+    private String type;
+    private String brand;
+    private String name;
+    private Double price;
+    private Double  availableSize;
+    private Double refreshRate;
+    private String screenType;
+    private String screenQuality;
+    private Boolean smartTv;
+    private Boolean wifi;
+    private Boolean voiceControl;
+    private Boolean hdr;
+    private Boolean bluetooth;
+    private Boolean ambiLight;
+    private Integer originalStock;
+    private Integer sold;
+    @ManyToOne
+    @JoinColumn(name = "cimodule_id")
```

Comment:
Deze joinCollumn name is bijna gelijk aan de name die automatische gegenereerd wordt (ci_module_id). Goed om zulke finetuning te gebruiken